integration_tests/jira_issues_tests.feature

```diff
@@ -121,6 +121,8 @@ Feature: Jira Issues Tests
     When we run the simulation with setup file json_file and parameters [24, 30, 30, 1]
     Then all load demands in setup was fulfilled on every market slot
 
+  # TODO: re-enable in context of bug/D3ASIM-3534
```

Comment:
Really curious why this only appeared now. The bug ticket should be prioritised.

Reply:
This bug has probably just showed itself because  the `jira_d3asim_1690` file has `ConstSettings.IAASettings.MARKET_TYPE = 2` + `ConstSettings.IAASettings.BID_OFFER_MATCH_TYPE = BidOfferMatchAlgoEnum.PAY_AS_CLEAR.value`
whereas in the changed src/d3a/d3a_core/sim_results/file_export_endpoints.py I have fixed the check 
`if ConstSettings.IAASettings.MARKET_TYPE == 3:`

Reply:
So basically it was there before and the export was broken without us noticing